bin/e2e

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I modeled this script off of https://github.com/forem/forem/blob/15d975d1220c2aeacf02021014ba9c83d5aee474/bin/knapsack_pro_rspec. Thanks for pointing me to this @mstruve!

Reply:
Below I noticed you are looking for the rspec token, even though technically either both will be there or they wont most of the time it would probably be a good idea to look for the Cypress token instead for the Cypress specs.

Reply:
I'll make the change. Good catch @mstruve! I forgot there was a separate token for Cypress/Knapsack integration.